source/_patterns/01-molecules/tag-list/tag-list.twig

```diff
@@ -1,4 +1,4 @@
-{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['tag-list']) }) -%}
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['tags']) }) -%}
```

Comment:
Matches `item-tags` (which isn't `item-tag-lists`), but a rename needs to be consistent. It's also independent of this change, so should be reverted.

Reply:
This has been reverted